src/opnsense/mvc/tests/app/config/config.php

```diff
@@ -31,11 +31,12 @@
 return new OPNsense\Core\AppConfig([
     'application' => [
         'controllersDir' => __DIR__ . '/../../../app/controllers/',
-        'modelsDir' => __DIR__ . '/../../../app/models/',
-        'viewsDir' => __DIR__ . '/../../../app/views/',
-        'pluginsDir' => __DIR__ . '/../../../app/plugins/',
-        'libraryDir' => __DIR__ . '/../../../app/library/',
-        'cacheDir' => __DIR__ . '/../../../app/cache/',
+        'modelsDir'      => __DIR__ . '/../../../app/models/',
```

Comment:
I have the feeling a style checker will revert this

Reply:
Done!